### PR TITLE
Layout & formatting of example descriptions on FHIR Artefacts page

### PR DESCRIPTION
**Pull request by dbojicic**
Changes to improve the example descriptions that are seen on the FHIR Artefacts page.
Improvement of layout and formatting example descriptions on FHIR Artefacts page.
This PR does not include in scope improving the content of the descriptions.

### CODE REVIEW

**Review by dt-r — CHANGES_REQUESTED** [review]
See inline comments noting that given the timeframes it may necessary change the framing of this PR to be layout, and move the project task to improve the descriptions into the backlog.

File: input/ig.xml [hunk 1]
```diff
@@ -221,23 +221,23 @@
 				<reference value="AllergyIntolerance/lactose"/>
 			</reference>
 			<name value="AllergyIntolerance - Lactose intolerance"/>
-			<description value="Lactose (food) intolerance&#xa;&#xa;Patient: Mia BANKS"/>
+			<description value="Shows an example for the *AU Core AllergyIntolerance* profile.&#xa;&#xa;Patient: Mia BANKS"/>
```
Comment (dt-r) [inline input/ig.xml:224]: Don't linebreak Patient. Apply to all example markdown.

Comment (dt-r) [inline input/ig.xml:224]: This description is insufficient in terms of providing information on the content.

File: input/ig.xml [hunk 2]
```diff
@@ -221,23 +221,23 @@
...
+			<description value="Shows an allergyintolerance example for the *AU Core AllergyIntolerance* indicating the patient has no known allergies.&#xa;&#xa;Patient: Mr. Li Wang"/>
 			<exampleCanonical value="http://hl7.org.au/fhir/core/StructureDefinition/au-core-allergyintolerance"/>
 		</resource>
 		<resource>
 			<reference>
 				<reference value="AllergyIntolerance/penicillin"/>
 			</reference>
 			<name value="AllergyIntolerance - Penicillin allergy"/>
-			<description value="Penicillin (medication) allergy&#xa;&#xa;Patient: Deangelo HOWE"/>
+			<description value="Shows an example for the *AU Core AllergyIntolerance* profile.&#xa;&#xa;Patient: Deangelo HOWE"/>
```
Comment (dt-r) [inline input/ig.xml:240]: This description is insufficient in terms of providing information on the content.

File: input/ig.xml [hunk 3]
```diff
@@ -246,31 +246,31 @@
 				<reference value="Condition/ckd"/>
 			</reference>
 			<name value="Condition - Chronic Kidney Disease"/>
-			<description value="Chronic Kidney Disease &#xa;&#xa;Patient: Mr. Harry Dan"/>
+			<description value="Shows an example of an active condition for a patient with an onset age for the *AU Core Condition* profile.&#xa;&#xa;Patient: Mr. Harry Dan"/>
```
Comment (dt-r) [inline input/ig.xml:249]: Don't need to say "for a patient".

File: input/ig.xml [hunk 4]
```diff
@@ -246,31 +246,31 @@
...
 			<exampleCanonical value="http://hl7.org.au/fhir/core/StructureDefinition/au-core-condition"/>
 		</resource>
 		<resource>
 			<reference>
 				<reference value="Condition/uti"/>
 			</reference>
 			<name value="Condition - Urinary tract infection"/>
-			<description value="Urinary tract infection&#xa;&#xa;Patient: Mia LEANNE BANKS"/>
+			<description value="Shows an example of a resolved condition for the *AU Core Condition* profile.&#xa;&#xa;Patient: Mia LEANNE BANKS"/>
```
Comment (dt-r) [inline input/ig.xml:257]: Missing info on the condition itself, and the other things the example demonstrates. Above says onset age, this has both onset and abatement.

File: input/ig.xml [hunk 5]
```diff
@@ -558,39 +558,39 @@
 				<reference value="Patient/ronny-irvine"/>
 			</reference>
 			<name value="Patient - Mr. Ronny Irvine (missing data for birth date)"/>
-			<description value="Mr. Ronny Irvine with missing data for birth date"/>
+			<description value="Shows a patient with missing data for birth date example for *AU Core Patient* profile."/>
```
Comment (dt-r) [inline input/ig.xml:561]: To be consistent with the MedicationRequest suggestion I made. This should be:

Shows an example of a Torres Strait Islander patient with DVA number for *AU Core Patient* profile illustrating missing data for birth date.

File: input/ig.xml [hunk 6]
```diff
@@ -558,39 +558,39 @@
...
 			<exampleCanonical value="http://hl7.org.au/fhir/core/StructureDefinition/au-core-patient"/>
 		</resource>
 		<resource>
 			<reference>
 				<reference value="Patient/bennelong-anne"/>
 			</reference>
 			<name value="Patient - Mrs. Anne Bennelong"/>
-			<description value="Mrs. Anne Bennelong"/>
+			<description value="Shows an example of a patient with a Medicare Number and communication preferences for the *AU Core Patient* profile."/>
```
Comment (dt-r) [inline input/ig.xml:569]: Should say at least.

Shows an example of an Aboriginal patient with Medicare Number who prefers to communicate in Yugambal for *AU Core Patient* profile.

File: input/ig.xml [hunk 7]
```diff
@@ -558,39 +558,39 @@
...
 			<exampleCanonical value="http://hl7.org.au/fhir/core/StructureDefinition/au-core-patient"/>
 		</resource>
 		<resource>
 			<reference>
 				<reference value="Patient/wang-li"/>
 			</reference>
 			<name value="Patient - Mr. Li Wang"/>
-			<description value="Mr. Li Wang"/>
+			<description value="Shows an example of a patient with an HPI-O scoped identifier and communications preferences for the *AU Core Patient* profile."/>
```
Comment (dt-r) [inline input/ig.xml:577]: Should say at least.

Shows an example of patient with an HPI-O scoped medical record number who prefers to communicate in Cantonese for *AU Core Patient* profile.

File: input/ig.xml [hunk 8]
```diff
@@ -599,31 +599,31 @@
 				<reference value="Practitioner/bobrester-bob"/>
 			</reference>
 			<name value="Practitioner - Dr. Bob Bobrester"/>
-			<description value="Dr. Bob Bobrester"/>
+			<description value="Shows a practitioner example for the *AU Core Practitioner* profile."/>
```
Comment (dt-r) [inline input/ig.xml:602]: Practitioner examples don't contain sufficient information.

File: input/ig.xml [hunk 9]
```diff
@@ -558,39 +558,39 @@
...
+			<description value="Shows an example of a patient with an HPI-O scoped identifier and communications preferences for the *AU Core Patient* profile."/>
 			<exampleCanonical value="http://hl7.org.au/fhir/core/StructureDefinition/au-core-patient"/>
 		</resource>
 		<resource>
 			<reference>
 				<reference value="Patient/banks-mia-leanne"/>
 			</reference>
 			<name value="Patient - Mia Leanne Banks"/>
-			<description value="Mia Leanne Banks"/>
+			<description value="Shows an example of *AU Core Patient* example that demonstrates the adopted extensions for sex and gender."/>
```
Comment (dt-r) [inline input/ig.xml:585]: Should say at least.

Shows an example of patient who has an IHI, identifies as female, whose sex at birth is recorded as female, identifies as she/her/her/hers/herself, and has multiple contact details for *AU Core Patient* profile.

File: input/ig.xml [hunk 10]
```diff
@@ -558,39 +558,39 @@
...
 			<exampleCanonical value="http://hl7.org.au/fhir/core/StructureDefinition/au-core-patient"/>
 		</resource>
 		<resource>
 			<reference>
 				<reference value="Patient/howe-deangelo"/>
 			</reference>
 			<name value="Patient - Deangelo HOWE"/>
-			<description value="Deangelo Howe"/>
```
Comment (dt-r) [inline input/ig.xml:593]: Should say at least.

Shows an example of patient who has an IHI and has multiple contact details for *AU Core Patient* profile.

File: input/ig.xml [hunk 11]
```diff
@@ -632,31 +632,31 @@
 				<reference value="PractitionerRole/bobrester-bob-gp"/>
 			</reference>
 			<name value="PractitionerRole - General practitioner"/>
-			<description value="Dr. Bob Bobrester, GP, at Bobrester Medical Centre"/>
+			<description value="Shows an example for the *AU Core PractitionerRole* profile with practitioner and organisation as references."/>
```
Comment (dt-r) [inline input/ig.xml:635]: here practitioner and organisation are lower case, in the following they are title case. I suspect this should be Practitioner and Organization.

File: input/ig.xml [hunk 12]
```diff
@@ -632,31 +632,31 @@
...
 			<exampleCanonical value="http://hl7.org.au/fhir/core/StructureDefinition/au-core-practitionerrole"/>
 		</resource>
 		<resource>
 			<reference>
 				<reference value="PractitionerRole/cardiologist-sallie-sutherland"/>
 			</reference>
 			<name value="PractitionerRole - Cardiologist"/>
-			<description value="Dr. Sallie Sutherland, Murrabit Public Hospital"/>
+			<description value="Shows an example for the *AU Core PractitionerRole* profile that demonstrates the use of Medicare Provider Number identifier and references to Practitioner, Organization and Location resources."/>
```
Comment (dt-r) [inline input/ig.xml:643]: remove the word "identifier" after "Medicare Provider Number" in all instances.

File: input/ig.xml [hunk 13]
```diff
@@ -665,15 +665,15 @@
 				<reference value="Procedure/dialysis"/>
 			</reference>
 			<name value="Procedure - Dialysis"/>
-			<description value="Dialysis &#xa;&#xa;Patient: Mr. Harry Dan"/>
+			<description value="Shows an example *AU Core Procedure*, with reason as a reference to a Condition resource.&#xa;&#xa;Patient: Ronny Lawrence Irvine"/>
```
Comment (dt-r) [inline input/ig.xml:668]: Procedures are all missing information like what the procedure is and other details.

File: input/ig.xml [hunk 14]
```diff
@@ -682,7 +682,7 @@
 				<reference value="Specimen/nasoswab"/>
 			</reference>
 			<name value="Specimen - Naso swab (COVID test)"/>
-			<description value="Naso swab specimen for COVID test&#xa;&#xa;Patient: Mr Li Wang"/>
+			<description value="This Specimen example is referenced by the *AU Core Pathology Result Observation* COVID test result example. &#xa;&#xa;Patient: Mr. Li Wang"/>
```
Comment (dt-r) [inline input/ig.xml:685]: Reformat. Shows a naso swab specimen referenced by the <insert hyperlink to the specific example> for the *AU Core Pathology Result Observation* profile. Patient: Mr. Li Wang.

**Review by dt-r — CHANGES_REQUESTED** [review]
See inline comments, noting given timeframes (this needs to be merged today) and priorities it may be better to:
* address only consistency and formatting.
* reframe this PR as layout & formatting rather than substantial content improvements to the descriptions.
* move the project task into the backlog to come back and improve these descriptions post ballot snapshot.

I will leave that to your discretion.

File: input/ig.xml (hunk 3 above)
Comment (dbojicic) [reply input/ig.xml:249]: Done

File: input/ig.xml (hunk 4 above)
Comment (dbojicic) [reply input/ig.xml:257]: Updated

File: input/ig.xml (hunk 1 above)
Comment (dbojicic) [reply input/ig.xml:224]: Done

File: input/ig.xml [hunk 15]
```diff
@@ -558,39 +558,39 @@
 				<reference value="Patient/ronny-irvine"/>
 			</reference>
 			<name value="Patient - Mr. Ronny Irvine (missing data for birth date)"/>
```
[the remark this answers is not in the document]
Comment (dbojicic) [reply input/ig.xml:560]: Fixed this

**Comment by dbojicic** [comment]
Moved tasked back to backlog (changed priority to P1). 
Changes made to:
- to use consistent format ` Shows an example of <insert thing here> for the *<insert profile>* profile. Patient: <insert patient>.`
- reference the right patient
- correct patient names and use consistent format

File: input/ig.xml [hunk 16]
```diff
@@ -246,31 +246,31 @@
...
+			<description value="Shows an example of a resolved condition for the *AU Core Condition* profile.&#xa;&#xa;Patient: Mia LEANNE BANKS"/>
 			<exampleCanonical value="http://hl7.org.au/fhir/core/StructureDefinition/au-core-condition"/>
 		</resource>
 		<resource>
 			<reference>
 				<reference value="Condition/condition-masked"/>
 			</reference>
 			<name value="Condition - Data suppressed for category and code"/>
-			<description value="Condition with data suppressed for category and code&#xa;&#xa;Patient: Anne Bennelong"/>
+			<description value="Shows an example for the *AU Core Condition* profile that demonstrates representiung supressed data for  category and code by using 'masked' from DataAbsentReason Code System. &#xa;&#xa;Patient: Anne Bennelong"/>
```
[the remark this answers is not in the document]
Comment (dt-r) [reply input/ig.xml]: Apply this change in this PR.

File: input/ig.xml [hunk 17]
```diff
@@ -599,31 +599,31 @@
...
+			<description value="Shows an example of a practitioner for the *AU Core Practitioner* profile."/>
 			<exampleCanonical value="http://hl7.org.au/fhir/core/StructureDefinition/au-core-practitioner"/>
 		</resource>
 		<resource>
 			<reference>
 				<reference value="Practitioner/sallie-sutherland"/>
 			</reference>
 			<name value="Practitioner - Dr. Sallie Sutherland"/>
-			<description value="Dr. Sallie Sutherland"/>
+			<description value="Shows an exaple of a practitioner with HPI-I for the *AU Core Practitioner* profile."/>
```
Comment (dt-r) [inline input/ig.xml:610]: "exaple" should be "example"

File: input/ig.xml [hunk 18]
```diff
@@ -599,31 +599,31 @@
...
 			<exampleCanonical value="http://hl7.org.au/fhir/core/StructureDefinition/au-core-practitioner"/>
 		</resource>
 		<resource>
 			<reference>
 				<reference value="Practitioner/chau-fryer"/>
 			</reference>
 			<name value="Practitioner - Dr. Chau FRYER"/>
-			<description value="Dr. Chau Fryer"/>
+			<description value="Shows an example of practitioner with HPI-I for the *AU Core Practitioner* profile."/>
```
Comment (dt-r) [inline input/ig.xml:618]: add "a" in front of "practitioner" in this and the following example for Darleen.

File: input/ig.xml [hunk 19]
```diff
@@ -632,31 +632,31 @@
 				<reference value="PractitionerRole/bobrester-bob-gp"/>
 			</reference>
 			<name value="PractitionerRole - General practitioner"/>
-			<description value="Dr. Bob Bobrester, GP, at Bobrester Medical Centre"/>
+			<description value="Shows an example of a General practitioner for the *AU Core PractitionerRole* profile with practitioner and organisation as references."/>
```
Comment (dt-r) [inline input/ig.xml:635]: Correct "General" to be "general"

File: input/ig.xml [hunk 20]
```diff
@@ -632,31 +632,31 @@
...
 			<exampleCanonical value="http://hl7.org.au/fhir/core/StructureDefinition/au-core-practitionerrole"/>
 		</resource>
 		<resource>
 			<reference>
 				<reference value="PractitionerRole/cardiologist-sallie-sutherland"/>
 			</reference>
 			<name value="PractitionerRole - Cardiologist"/>
-			<description value="Dr. Sallie Sutherland, Murrabit Public Hospital"/>
+			<description value="Shows an example of a Cardiologist for the *AU Core PractitionerRole* profile that demonstrates the use of Medicare Provider Number and practitioner, organization and location references."/>
```
Comment (dt-r) [inline input/ig.xml:643]: 1. "Cardiologist" to be "cardiologist"
2. Title case the FHIR resources.

File: input/ig.xml (hunk 14 above)
Comment (dt-r) [reply input/ig.xml]: Due to markdown formatting half my comment disappeared. It was meant to be:
`Reformat. Shows a naso swab specimen referenced by the <insert hyperlink to the specific example> for the *AU Core Pathology Result Observation* profile. Patient: Mr. Li Wang.`

Please adjust now that I've made the full text of the comment visible.

File: input/ig.xml [hunk 21]
```diff
@@ -396,135 +396,135 @@
...
+			<description value="Shows a blood pressure example for the *AU Core Blood Pressure* profile, including both systolic and diastolic measurements.&#xa;&#xa;Patient: Mr. Li Wang"/>
 			<exampleCanonical value="http://hl7.org.au/fhir/core/StructureDefinition/au-core-bloodpressure"/>
 		</resource>
 		<resource>
 			<reference>
 				<reference value="Observation/bloodpressure-data-absent"/>
 			</reference>
 			<name value="Observation - Blood Pressure (data absent for diastolic)"/>
-			<description value="Blood Pressure - data absent for diastolic&#xa;&#xa;Patient: Mr. Harry Dan"/>
+			<description value="This is a blood pressure data absent example for the *AU Core Blood Pressure* profile, demonstrating how to represent blood pressure with a missing diastolic measurement.&#xa;&#xa;Patient: Mr. Harry Dan"/>
```
[the remark this answers is not in the document]
Comment (dt-r) [reply input/ig.xml]: Apply this change in this PR.

File: input/ig.xml [hunk 22]
```diff
@@ -396,135 +396,135 @@
...
+			<description value="Shows an HDL result example for the *AU Core Pathology Result Observation* profile.&#xa;&#xa;Patient: Mia LEANNE BANKS"/>
 			<exampleCanonical value="http://hl7.org.au/fhir/core/StructureDefinition/au-core-diagnosticresult-path"/>
 		</resource>
 		<resource>
 			<reference>
 				<reference value="Observation/lipid-ldl-1"/>
 			</reference>
 			<name value="Observation - LDL"/>
-			<description value="LDL result&#xa;&#xa;Patient: Mia LEANNE BANKS"/>
+			<description value="Shown an LDL result example for the *AU Core Pathology Result Observation* profile.&#xa;&#xa;Patient: Mia LEANNE BANKS"/>
```
[the remark this answers is not in the document]
Comment (dt-r) [reply input/ig.xml]: Apply this change in this PR.

File: input/ig.xml [hunk 23]
```diff
@@ -396,135 +396,135 @@
...
+			<description value="Shows a Triglyceride result example for the *AU Core Pathology Result Observation* profile.&#xa;&#xa;Patient: Ms. Mia LEANNE BANKS"/>
 			<exampleCanonical value="http://hl7.org.au/fhir/core/StructureDefinition/au-core-diagnosticresult-path"/>
 		</resource>
 		<resource>
 			<reference>
 				<reference value="Observation/pathresult-covid-1"/>
 			</reference>
 			<name value="Observation - COVID Pathology Result (Not detected)"/>
-			<description value="COVID Pathology Result (Not detected)&#xa;&#xa;Patient: Mr. Li Wang"/>
+			<description value="Shows a COVID Pathology Result (with an HPI-O scoped identifier) example for the *AU Core Pathology Result Observation* profile. &#xa;&#xa;Patient: Mr. Li Wang"/>
```
[the remark this answers is not in the document]
Comment (dt-r) [reply input/ig.xml]: Apply these parts of the comment in this PR:
1. HPI-O scoped what? practitioner? report? order?
2. Don't need to say "COVID Pathology Result" , just say the test name.

File: input/ig.xml [hunk 24]
```diff
@@ -396,135 +396,135 @@
...
 			<exampleCanonical value="http://hl7.org.au/fhir/core/StructureDefinition/au-core-diagnosticresult-path"/>
 		</resource>
 		<resource>
 			<reference>
 				<reference value="Observation/resprate-1"/>
 			</reference>
 			<name value="Observation - Respiration rate"/>
-			<description value="Observation of respiration rate&#xa;&#xa;Patient: Mia LEANNE BANKS"/>
+			<description value="Shows an observation of respiration rate example for the *AU Core Respiration Rate* profile. &#xa;&#xa;Patient: Mia LEANNE BANKS"/>
```
[the remark this answers is not in the document]
Comment (dt-r) [reply input/ig.xml]: Apply this change in this PR.

File: input/ig.xml (hunk 9 above)
Comment (dt-r) [reply input/ig.xml]: This still needs to be reformatted so it's consistent.

**Review by dt-r — CHANGES_REQUESTED** [review]
See inline comments & unresolved conversations. You will need to unhide the hidden conversations to see all unresolved conversations.

File: input/ig.xml (hunk 16 above)
[the remark this answers is not in the document]
Comment (dbojicic) [reply input/ig.xml]: Applied.

File: input/ig.xml (hunk 21 above)
[the remark this answers is not in the document]
Comment (dbojicic) [reply input/ig.xml]: Applied.

File: input/ig.xml (hunk 22 above)
[the remark this answers is not in the document]
Comment (dbojicic) [reply input/ig.xml]: Fixed.

File: input/ig.xml (hunk 23 above)
[the remark this answers is not in the document]
Comment (dbojicic) [reply input/ig.xml]: Updated

File: input/ig.xml (hunk 24 above)
[the remark this answers is not in the document]
Comment (dbojicic) [reply input/ig.xml]: Applied

File: input/ig.xml (hunk 9 above)
Comment (dbojicic) [reply input/ig.xml]: Applied.

File: input/ig.xml (hunk 14 above)
Comment (dbojicic) [reply input/ig.xml]: Applied

File: input/ig.xml (hunk 19 above)
Comment (dbojicic) [reply input/ig.xml:635]: Changed

Comment (dbojicic) [reply input/ig.xml:635]: Changed

File: input/ig.xml (hunk 20 above)
Comment (dbojicic) [reply input/ig.xml:643]: Done

File: input/ig.xml (hunk 18 above)
Comment (dbojicic) [reply input/ig.xml]: Done

File: input/ig.xml (hunk 17 above)
Comment (dbojicic) [reply input/ig.xml]: Changed

File: input/ig.xml [hunk 25]
```diff
@@ -396,135 +396,135 @@
...
 			<exampleCanonical value="http://hl7.org.au/fhir/core/StructureDefinition/au-core-waistcircum"/>
 		</resource>
 		<resource>
 			<reference>
 				<reference value="Observation/observation-masked"/>
 			</reference>
-			<name value="Observation - Data suppressed for category, code and effectiveDate"/>
-			<description value="Observation with masked category, code and effectiveDate&#xa;&#xa;Patient: Mr. Ronny Irvine"/>
+			<name value="Observation - Data suppressed for category, code, effective[x] and value[x]"/>
+			<description value="Shows an example of suppressed data (category, code, effectivePeriod and valueQuantity) for the *AU Core Diagnostic Result Observation* profile. Patient: Ronny Lawrence Irvine."/>
```
Comment (dt-r) [inline input/ig.xml:527]: The suppressed data change is not applied here yet.

Should say: Shows an example of an observation with suppressed data (category, code, effectivePeriod, and valueQuantity) where the data requester has access rights to know that data is suppressed for the *AU Core Diagnostic Result Observation* profile. Patient: Ronny Lawrence Irvine.

File: input/ig.xml [hunk 26]
```diff
@@ -396,135 +396,135 @@
...
+			<description value="Shows an example of a Triglyceride result for the *AU Core Pathology Result Observation* profile. Patient: Mia Leanne Banks."/>
 			<exampleCanonical value="http://hl7.org.au/fhir/core/StructureDefinition/au-core-diagnosticresult-path"/>
 		</resource>
 		<resource>
 			<reference>
 				<reference value="Observation/pathresult-covid-1"/>
 			</reference>
 			<name value="Observation - COVID Pathology Result (Not detected)"/>
-			<description value="COVID Pathology Result (Not detected)&#xa;&#xa;Patient: Mr. Li Wang"/>
+			<description value="Shows an example of a COVID-19 PCR test result indicating 'Not detected' for the *AU Core Diagnostic Result - Pathology* profile, with notes on test accreditation and evaluation status. Patient: Li Wang."/>
```
Comment (dt-r) [inline input/ig.xml:495]: reorder the words so the profile is at the end. Should say:

> Shows an example of a COVID-19 PCR test result indicating 'Not detected' with notes on test accreditation and evaluation status for the *AU Core Diagnostic Result - Pathology* profile. Patient: Li Wang.

File: input/ig.xml [hunk 27]
```diff
@@ -682,7 +682,7 @@
 				<reference value="Specimen/nasoswab"/>
 			</reference>
 			<name value="Specimen - Naso swab (COVID test)"/>
-			<description value="Naso swab specimen for COVID test&#xa;&#xa;Patient: Mr Li Wang"/>
+			<description value="Shows a naso swab specimen referenced by the [COVID Pathology Result (Not detected)](Observation-pathresult-covid-1.html) example for the *AU Core Pathology Result Observation* profile. Patient: Mr. Li Wang."/>
```
Comment (dt-r) [inline input/ig.xml:685]: Title of the example is `[Observation - COVID Pathology Result (Not detected)](https://build.fhir.org/ig/hl7au/au-fhir-core/Observation-pathresult-covid-1.html)`. Please correct

**Review by dt-r — CHANGES_REQUESTED** [review]
See inline comments.

File: input/ig.xml (hunk 25 above)
Comment (dbojicic) [reply input/ig.xml:527]: Done, sorry - missed this one earlier

File: input/ig.xml (hunk 26 above)
Comment (dbojicic) [reply input/ig.xml:495]: Done

File: input/ig.xml (hunk 27 above)
Comment (dbojicic) [reply input/ig.xml:685]: Done

**Review by dt-r — APPROVED** [review]

**Pull request merged**